src/pages/explore/BiddersPage/BidVouch.tsx

```diff
@@ -82,7 +91,50 @@ const BidVouch = ({ handleResult, activeAccount } : BidVouchProps) => {
           </div>
         </Tab.Pane>
         <Tab.Pane eventKey="vouch">
-          Vouch
+          <Form onSubmit={handleVouchSubmit}>
+            <Form.Group className="mb-3">
+              <StyledFormLabel>Vouch for</StyledFormLabel>
+              <StyledSelectForm aria-label="Default select example">
+                <option>Select an account</option>
+                {accounts.map((account, key) => <option key={key} value={account.address}>{account.name}</option>)}
+              </StyledSelectForm>
+            </Form.Group>
+
+            <Form.Group className="mb-3">
+              <StyledFormLabel>Bid amount</StyledFormLabel>
+              <StyledFormInput className="mb-3">
+                <StyledForm
+                  type="number"
+                  step="any"
+                  placeholder="0.0000"
+                  aria-label="Bid amount"
+                />
+                <StyledInputGroupText>KSM</StyledInputGroupText>
+              </StyledFormInput>
+            </Form.Group>
+            <Form.Group className="mb-3">
+              <StyledFormLabel>Tip amount</StyledFormLabel>
+              <StyledFormInput className="mb-3">
+                <StyledForm
+                  type="number"
+                  step="any"
+                  placeholder="0.0000"
+                  aria-label="Tip amount"
+                />
+                <StyledInputGroupText>KSM</StyledInputGroupText>
+              </StyledFormInput>
+            </Form.Group>
+            <Button disabled={loading} variant="primary" type="submit" className="w-100">
+              {loading ? <Spinner size="sm" animation="border" /> : 'Submit'}
+            </Button>
+            <StyledButtonLabel className="text-muted">
+              *Plus 0.0045 KSM fee
```

Comment:
Can we fetch this value from the api instead of hardcoding it? This way we'll most likely never have to update this code and it also guarantees users won't see this fee and be charged other value.

Reply:
This is a great point. I think we should do it, but in another PR.